nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -135,7 +135,6 @@ public NullnessHint onDataflowVisitMethodInvocation(
   }
 
   private void setConditionalArgumentNullness(
-      AccessPathNullnessPropagation.Updates thenUpdates,
```

Comment:
Passing the `elseUpdates` without the `thenUpdates` is a bit weird, even if we are only changing the later at the moment.

Reply:
It's an unused variable. Can put it back and suppress if need be?

Reply:
Going back and forth on this one, but I think it's fine. I am ok with this change removing the unused `thenUpdates`. Even thought a method taking only the `elseUpdates` is a bit weird, that's what the method actually does currently, regardless of the formal list, so...

Reply:
👍